tests/conftest.py

```diff
@@ -1,11 +1,31 @@
 import asyncio
+import uuid
+from distutils.version import StrictVersion
+from os import environ as ENV
 
 import pytest
 
 from aiodocker.docker import Docker
 from aiodocker.exceptions import DockerError
 
 
+_api_versions = {
```

Comment:
I don't understand. That's exactly what this PR does right? Or you mean using a different API version for the same Docker version? I think that would be overkill.